### PR TITLE
[#175] Document exposing the projectId as the environment variable CY…

### PR DESCRIPTION
**Pull request by richdouglasevans**
Addresses #175 Mention `CYPRESS_PROJECT_ID`

mention-env

### CODE REVIEW

File: source/guides/core-concepts/dashboard-service.md [hunk 1]
```diff
@@ -110,7 +110,13 @@ Once you set up your project to record, we generate a unique `projectId` for you
...
 ```
 
-This helps us uniquely identify your project. If you manually alter this, **Cypress will no longer be able to identify your project or find the recorded builds for it**. We recommend that you check your `cypress.json` including the `projectId` into source control.
+This helps us uniquely identify your project. If you manually alter this, **Cypress will no longer be able to identify your project or find the recorded builds for it**.
+
+We recommend that you check your `cypress.json` including the `projectId` into source control. If you don't want your `projectId` visible in your source code you can set it as an environment variable from your CI provider using the name `CYPRESS_PROJECT_ID`. The exact mechanism for doing so depends on your CI provider but could be as simple as:
+
+```bash
+$ export CYPRESS_PROJECT_ID={projectId}   # replace {projectId} with your actual projectId
+```
```
Comment (richdouglasevans) [inline source/guides/core-concepts/dashboard-service.md]: I believe that this change addresses the specific thrust of Gleb's ticket.

It's in the docs for the dashboard as opposed to the FAQ following a steer from Jennifer about where best to put this. (See the comments in #175.)

mention-env

File: source/faq/questions/using-cypress-faq.md [hunk 2]
```diff
@@ -311,35 +311,25 @@ It is possible to upload files in your application but it's different based on h
 
 ## {% fa fa-angle-right %} What is the projectId for?
 
-Once you {% url "set up your tests to record" dashboard-service#Recorded-runs %}, we generate a unique `projectId` for your project and automatically insert it into your `cypress.json` file.
-
-***The `projectId` is a 6 character string in your cypress.json:***
+The `projectId` is a 6 character string that helps identify your project once you've {% url "set up your tests to record" dashboard-service#Recorded-runs %}. It's generated by Cypress and typically is found in your {% url `cypress.json` configuration %} file.
```
Comment (richdouglasevans) [inline source/faq/questions/using-cypress-faq.md]: > This is a slight expansion to the scope of the original ticket.

I have tried to preserve the answer the FAQ question while also addressing (removing) the duplication of content in this section and what's already documented in the _Dashboard Services_ page.

Someone reading the FAQ should still be able to grok what the `projectId` is from this smaller answer. The answer directs them to the fuller documentation if they wanna dig in.

from-faq

File: source/faq/questions/using-cypress-faq.md [hunk 3]
```diff
@@ -311,35 +311,25 @@ It is possible to upload files in your application but it's different based on h
...
 
-This is how we uniquely identify your project. If you manually alter this, *Cypress will no longer be able to identify your project or find the recorded tests for it*. We recommend that you check your `cypress.json`, including the `projectId`, into source control.
+For further detail see the {% url Identification dashboard-service#Identification %} section of the {% url "Dashboard Service" dashboard-service %} docs.
 
 ## {% fa fa-angle-right %} What is a Record Key?
 
-Once you're {% url "set up to record test runs" dashboard-service#Recorded-runs %}, we automatically generate a *Record Key* for the project.
-
-***A record key is a GUID that looks like this:***
+A *Record Key* is a GUID that's generated automatically by Cypress once you've {% url "set up your tests to record" dashboard-service#Recorded-runs %}. It helps identify your project and authenticate that your project is even *allowed* to record tests.
```
Comment (richdouglasevans) [inline source/faq/questions/using-cypress-faq.md]: This FAQ also duplicated the existing documentation from the _Dashboard Services_ page.

Again, I've tried to answer the question with a direct answer and a screenshot of where to find the _Record Key_ while also removing much of the duplication and directing them to the fuller documentation if they wanna dig in.

In making these changes I asked myself _"If I wanted to know what the `projectId`/Record Key were, would I be satisfied with these answers?"_... and yeah, I would. School me if they're too brief and/or don't work for you.

Comment (jennifer-shehane) [reply source/faq/questions/using-cypress-faq.md]: Great, yes, this is preferable to what we had.

**Pull request merged**